substra/sdk/schemas.py

```diff
@@ -169,8 +170,8 @@ def build_request_kwargs(self, local):
 def check_asset_key_or_parent_ref(cls, values):
     """Check that either (asset key) or (parent_task_key, parent_task_output_identifier) are set, but not both."""
 
-    has_asset_key = bool(values.get("asset_key"))
-    has_parent = bool(values.get("parent_task_key")) and bool(values.get("parent_task_output_identifier"))
+    has_asset_key = bool(dict(values).get("asset_key"))
```

Comment:
what is the type of `values`? 
is there maybe another way to get an item out of this object?

Reply:
I tried but the type of values is not stable. Sometimes its already a dict, sometimes not, so its the best way I found to keep the same behaviour :/